research/object_detection/core/target_assigner.py

```diff
@@ -1897,3 +1911,197 @@ def assign_corner_offset_targets(
 
     return (tf.stack(corner_targets, axis=0),
             tf.stack(foreground_targets, axis=0))
+
+class DETRTargetAssigner(object):
```

Comment:
I am still a bit worried that we are carrying around a lot of legacy code for the DETR Target Assigner. Can you take a look at  https://github.com/facebookresearch/detr/blob/master/models/matcher.py  and see if we can do something similiar ?

Reply:
That file just does the matching. Do you mean the fact that we do similarity calculation separately from the target assigner?